templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -199,17 +199,25 @@ objects:
             enabled: ${ENABLE_BASIC_AUTH}
 
 - apiVersion: servicecatalog.k8s.io/v1alpha1
-  kind: Broker
+  kind: ${BROKER_KIND}
   metadata:
     name: ansible-service-broker
   spec:
     url: ${ASB_SCHEME}://asb-1338-ansible-service-broker.${ROUTING_SUFFIX}
     authInfo:
-      basicAuthSecret:
-        namespace: ansible-service-broker
-        name: asb-auth-secret
+      ${{BROKER_AUTH}}
```

Comment:
Thanks @fabianvf for this magic.

Reply:
haha. I was trying to figure out how to make this work and didn't find this gem.